plugins/azure/mysqlserver/mysqlFlexibleServerCMKEncrypted.js

```diff
@@ -0,0 +1,53 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'MySQL Flexible Server CMK Encrypted',
+    category: 'MySQL Server',
+    domain: 'Databases',
+    severity: 'High',
+    description: 'Ensures that MySQL flexible servers are encrypted using CMK.',
```

Comment:
https://learn.microsoft.com/en-us/azure/mysql/flexible-server/concepts-customer-managed-key as per link its Data encryption with customer managed keys for Azure Database for MySQL  can we update the plugin accordingly, the description, messages etc the name can be **MySQL Flexible Server Data CMK Encrypted** or **MySQL Flexible Server Encryption At Rest with BYOK**